src/app/containers/RadioPageBlocks/Blocks/OnDemandImage/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useContext } from 'react';
+import { string } from 'prop-types';
+import styled from 'styled-components';
+import Image, { AmpImg } from '@bbc/psammead-image';
+import {
+  GEL_SPACING_QUAD,
+  GEL_SPACING_DBL,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_1_SCREEN_WIDTH_MAX } from '@bbc/gel-foundations/breakpoints';
+import { RequestContext } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+const ImageContainer = styled.div`
+  padding: ${GEL_SPACING_QUAD}
+    ${({ dir }) => (dir === 'ltr' ? GEL_SPACING_DBL : 0)} ${GEL_SPACING_QUAD}
```

Comment:
do we need to do this sort of thing? I thought we have something in place that handles this? Or maybe this is when I worked in webcore 🤔